packages/visx-shape/src/shapes/BarRounded.tsx

```diff
@@ -35,34 +35,50 @@ export type BarRoundedProps = {
   bottomLeft?: boolean;
   /** apply corner radius to bottom right */
   bottomRight?: boolean;
+  /** Optional children override. */
+  children?: ({ path }: { path: string }) => React.ReactNode;
 };
 
-export default function BarRounded({
-  className,
-  innerRef,
-  x,
-  y,
-  width,
+/** Hook that returns a BarRounded path. */
```

Comment:
The name `useXXX` misled me a bit. I thought this is a `react` hook but it just returns a path string.
Could go with `createBarRoundedPath()` for naming as well.

Reply:
the idea is that it could be used as a hook, hence the `useXXX`, I'll update to `useBarRoundedPath` tho I agree that's clearer.